src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+            while (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                setManufacturer(manufacturer, resultSet);
+            }
+            return Optional.of(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer by id " + id + " from DB", e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        String getAllRequest = "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturersStatement =
+                        connection.prepareStatement(getAllRequest)) {
+            ResultSet resultSet = createManufacturersStatement.executeQuery();
+            while (resultSet.next()) {
+                Manufacturer manufacturer = new Manufacturer();
+                setManufacturer(manufacturer, resultSet);
+                allManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturers SET country = ? "
+                + "WHERE is_deleted = FALSE AND id = ?";
+        Manufacturer updatedManufacturer = new Manufacturer();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(updateRequest)) {
+            createManufacturerStatement.setLong(2, manufacturer.getId());
+            createManufacturerStatement.setString(1, manufacturer.getCountry().toUpperCase());
+            createManufacturerStatement.executeUpdate();
+            return updatedManufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer " + manufacturer, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteManufacturerRequest = "UPDATE manufacturers SET is_deleted = TRUE "
+                + "WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(deleteManufacturerRequest)) {
+            createManufacturerStatement.setLong(1, id);
+            return createManufacturerStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete manufacturer by id " + id, e);
+        }
+    }
+
+    private void setManufacturer(Manufacturer manufacturer, ResultSet resultSet) {
```

Comment:
```suggestion
    private Manufacturer getManufacturer(ResultSet resultSet) {
```

let's return manufactirer from this method